core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/experiments-list/experiments-list.component.html

```diff
@@ -0,0 +1 @@
+<p>experiments-list works!</p>
```

Comment:
i18n?

Reply:
Added DotMessagePipeModule to the module.